client/state/selectors/is-eligible-for-upwork-support.ts

```diff
@@ -12,22 +12,22 @@ import { getCurrentUserLocale } from 'state/current-user/selectors';
 import getSitesItems from 'state/selectors/get-sites-items';
 import { isBusinessPlan, isEcommercePlan } from 'lib/plans';
 
-const SPANISH_LANG_SLUGS = [ 'es', 'es-cl', 'es-mx' ];
+const UPWORK_LOCALES = [ 'es', 'es-cl', 'es-mx', 'fr', 'fr-ca', 'fr-be', 'fr-ch' ];
 
 /**
  * @param {Object} state Global state tree
  * @return {Boolean} Whether or not this customer should receive Upwork support
  */
-export default function isEligibleForUpworkSupport( state ) {
-	// Upwork is currently available for Spanish-speaking users only
-	if ( ! includes( SPANISH_LANG_SLUGS, getCurrentUserLocale( state ) ) ) {
+export default function isEligibleForUpworkSupport( state ): boolean {
+	if ( ! includes( UPWORK_LOCALES, getCurrentUserLocale( state ) ) ) {
 		return false;
 	}
 
 	const hasBusinessOrEcommercePlan = some( getSitesItems( state ), site => {
 		const planSlug = get( site, 'plan.product_slug' );
 		return isBusinessPlan( planSlug ) || isEcommercePlan( planSlug );
 	} );
+
 	// Upwork is not available if the customer has a Business or eCommerce plan
-	return ! hasBusinessOrEcommercePlan;
+	return hasBusinessOrEcommercePlan ? false : true;
```

Comment:
haha! type error resolved!

at times I want to say "no! you are wrong TS" but then it's always right…

Reply:
oh wait, why was this wrong? won't the `!` immediately turn this into a `boolean`?

Reply:
It wasn't wrong. I tweaked it for personal preference readability reasons. I can put it back if anyone cares particularly.

(I still don't love it with `false` first -- not very natural.  Ideally, to me, the return would be something like `return hasUpworkEligiblePlan` but I didn't want to mess with existing logic that much.)

Reply:
> `return hasUpworkEligiblePlan`

that sounds more semantic doesn't it?

for what it's worth, the `return condition ? false : true` feels awkward to me and only exists in a few places in Calypso, almost never when `condition` is already boolean

Reply:
Reverted.